docs/how-to/installation/docker.txt

```diff
@@ -34,16 +33,57 @@ Preliminary Setup
       docker pull determinedai/determined-master:VERSION
       docker pull determinedai/determined-agent:VERSION
 
+#. Assuming you are running multiple containerized services on the Determined
+   master, optionally create the ``determined`` network that we will use in the
+   service startup commands below:
+
+   .. code::
+
+      docker network create determined
+
+
 Configuring and Starting the Cluster
 ------------------------------------
 
-Determined Master and Agents
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+PostgreSQL and Hasura
+~~~~~~~~~~~~~~~~~~~~~
+The following commands start PostgreSQL and Hasura in containers on the master
+and use the ``determined`` network created above.
+
+Start PostgreSQL on the master:
```

Comment:
non-blocking: Can probably remove this and the next instance of "on the master", since it's clear from context.